tests/nightly/JenkinsfileForBinaries

```diff
@@ -106,6 +106,14 @@ core_logic: {
           utils.docker_run('ubuntu_nightly_gpu', 'nightly_tutorial_test_ubuntu_python3_gpu', true)
         }
       }
+    },
+    'Clojure Nightly: CPU': {
+      node(NODE_LINUX_GPU) {
+        ws('workspace/clojure') {
+          utils.unpack_and_init('gpu', mx_lib)
+          utils.docker_run('ubuntu_nightly_gpu', 'unittest_ubuntu_cpu_clojure_integration', false)
```

Comment:
unittest_ubuntu_cpu_clojure_integration  Are these CPU or GPU? maybe rename the function to XPU?

Reply:
@larroy  Merci ! Thanks for the comment ! Right now and as it is, only CPU but it's almost ready to run a few GPU tests  
I don't see any reference to XPU in master, is another language already using the notation ?

Reply:
In c++ we use the notion for operator definitions that are applicable for CPU as well as GPU, thus called XPU. I can look for a reference if you're interested